prompt_lean_setup

```diff
@@ -67,13 +71,15 @@ prompt_lean_human_time() {
 
 # fastest possible way to check if repo is dirty
 prompt_lean_git_dirty() {
-    # check if we're in a git repo
-    command git rev-parse --is-inside-work-tree &>/dev/null || return
-    # check if it's dirty
-    local umode="-uno" #|| local umode="-unormal"
-    command test -n "$(git status --porcelain --ignore-submodules ${umode} 2>/dev/null | head -100)"
-
-    (($? == 0)) && echo ' +'
+    if [[ $PROMPT_LEAN_NOVCS != 1 ]]; then
```

Comment:
you can just check the value and then return and end the if-then. the rest of the code can then moved of the then part.